src/calliope/config/model_data_checks.yaml

```diff
@@ -29,6 +29,15 @@ fail:
   - where: storage_initial>1
     message: "storage_initial is a fraction; values larger than 1 are not allowed."
 
+  - where: storage_initial<0
```

Comment:
merge with above: `where: storage_initial<0 or storage_initial>1`

Reply:
Fixed in 576c13e1a585702d08c1332bb192a0e62b4378e0